lib/fetch/request.js

```diff
@@ -138,8 +138,47 @@ class Request {
 
     // 12. Set request to a new request with the following properties:
     request = makeRequest({
-      ...request,
-      window
+      // URL request’s URL.
```

Comment:
this was previously defined as the [current url](https://fetch.spec.whatwg.org/#concept-request-current-url) which is the last url in the urlList, but the recent spec changes technically fixed this on behalf of undici :)